backend/game/turn_actions/test_heal.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import game.turn_actions.heal as heal
+from game.player.player import Player
+
+
+@pytest.fixture(autouse=True)
+def hurt_player():
+    hurt_player: Player = Player()
+    hurt_player.update_health_by(-3)
+    return hurt_player
+
+
+def test_heal_from_dice_out_of_tokyo(hurt_player):
```

Comment:
I'd like to see both of these tests exist PLUS:
The same 2 tests, but actually hitting the dice resolver method, to assert that the dice resolver uses this methods and behaves correctly.